en/reports/vivliostyle-report-2022/report.css

```
@@ -0,0 +1,121 @@
+﻿@page {
+  /* size: a4; */
+  margin: 10%;
+  font-family: "Helvetica Neue", "Helvetica", "Arial", "YuGothic", "Yu Gothic", sans-serif;
+  font-size: 9pt;
+  line-height: 1;
+
+  @top-center {
+    content: env(doc-title);
+  }
+
+  @bottom-center {
+    content: counter(page);
+  }
+}
+
+@page :first {
+  @top-center {
+    content: none;
+  }
+}
+
+html {
+  ffont-family: "Optima", "Arial", sans-serif;
+}
+
+body {
+  font-size: 94%;
+  line-height: 1.5;
+}
+
+@media print {
+  body {
+    column-width: 34em;
+    column-gap: 1.5em;
+    column-fill: auto;
+  }
+}
+
+h1.title {
+  margin: 1.5em 0 1em;
+  font-size: 270%;
+  line-height: 1.5;
+  text-align: center;
+  word-break: keep-all;
+  overflow-wrap: break-word;
+}
+
+#title-block-header::after {
+  content: url(https://vivliostyle.org/assets/vivliostyle-logo.svg);
+  display: block;
+  width: min-content;
+  margin: 3em auto 5em;
+}
+
+#TOC>ul>li>a {
+  font-size: 110%;
+  font-weight: bold;
+}
+
+p {
+  text-align: justify;
+}
+
+h2 {
+  margin: 0 0 3.3em;
+  font-size: 133%;
+  line-height: 1.5;
+}
+
+h3 {
+  margin: 1.5 0 1.5em;
+  font-size: 120%;
+  line-height: 1.5;
+}
+
+h2 {
+  break-before: page;
+}
+
+/*
+h2 ~ h2 ~ h3 ~ h3 {
+  break-before: column;
+}
+ */
+
```

生成されたPDFを見て、p.9〜p.10でTable-1が分割されているのが気になりました。このTableはfigure要素で囲んでます。ほかのfigure要素も1ページ内に収めるべきものです。そこでfigureにbreak-inside:avoidを指定して分割されないようにするのがよさそうです。

```suggestion

figure {
  break-inside: avoid;
}

```